keyboards/sharkoon/skiller_sgk50_s4/keyboard.json

```diff
@@ -21,7 +21,7 @@
         "rgb_matrix": true
     },
     "matrix_pins": {
-        "cols": ["C0", "C1", "C2", "C3", "A6", "B1", "B10", "B11", "B12", "B13", "B14", "B15", "C6", "C7"],
+        "cols": ["C0", "C1", "C2", "C3", "A6", "B1", "B10", "B11", "B12", "B13", "B14", "C6", "C7", "C8", "C9", "A8"],
```

Comment:
This appears to be a significant change (B15 is dropped, and 3 additional pins added)